utils/src/main/scala/za/co/absa/enceladus/utils/fs/FileSystemUtils.scala

```diff
@@ -46,6 +49,23 @@ object FileSystemUtils {
     }
   }
 
+  implicit class FileSystemExt(fs: FileSystem) {
+    /**
+     * Checks if path is a directory, returns `default` on FileNotFoundException. Basically, behaves the same way as the late
+     * [[org.apache.hadoop.fs.FileSystem#isDirectory(org.apache.hadoop.fs.Path)]]
+     * @param path path to test if isDirectory
+     * @param default default to be returned on error
+     */
+    def isDirectoryWithDefault(path: Path, default: Boolean): Boolean = {
```

Comment:
This function is implemented twice. Can we remove one of them?

Reply:
Probably the one in EnceladusFileSystem?

Reply:
Nice catch

Reply:
Will do as part of #2158